examples/index.tsx

```diff
@@ -1217,10 +1217,6 @@ const ToggleSwitchLabelsExample = withThemeProvider(
 );
 export { ToggleSwitchLabelsExample };
 
-import { default as ToggleSwitchIconExampleRaw } from './ToggleSwitch.icon';
-const ToggleSwitchIconExample = withThemeProvider(ToggleSwitchIconExampleRaw);
```

Comment:
(nit): I think that it might be better to replace this example with one that uses a custom icon instead of completely removing it. I know that we are going to look into deprecating that functionality, but we still have it available currently.

Reply:
The reasoning is mentioned in the PR description:

> Removed the "With icon" section from docs. The main use case for `icon` is to add a checkmark; **we shouldn't encourage adding a custom icon**.

Reply:
Adding a custom icon can be discouraged without completely removing the example. There could be a highlighted text box warning users about the issues that could be caused by using a custom icon. Not a big deal either way.